src/pulse3D/plate_recording.py

```diff
@@ -359,14 +358,16 @@ def __init__(
             with tempfile.TemporaryDirectory() as tmpdir:
                 zf = zipfile.ZipFile(path)
                 zf.extractall(path=tmpdir)
-                self.wells, calibration_recordings = load_files(
-                    tmpdir, stiffness_factor, inverted_post_magnet_wells
-                )
+
+                if glob.glob(os.path.join(tmpdir, "**", "*.h5"), recursive=True):
```

Comment:
Should we enforce the zip files to only contains files?

Reply:
like instead of the `else` do `elif glob.glob(os.path.join(tmpdir, "*.xlsx"), recursive=True)`